zerver/migrations/0281_remove_night_mode_add_color_scheme.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.12 on 2020-05-15 14:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0280_userprofile_presence_enabled'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='userprofile',
+            name='night_mode',
+        ),
+        migrations.AddField(
+            model_name='userprofile',
+            name='color_scheme',
+            field=models.PositiveSmallIntegerField(default=1),
+        ),
+    ]
```

Comment:
Usually, I would suggest that we 

BTW we should set `AUTOMATIC_THEME_DETECTION = 1` and use that variable in this migration.

Reply:
I think the best answer is probably have the migration preserve `night_mode` (the old default should turn into the new default of automatic).